src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxProcessingDaoJdbc.java

```diff
@@ -131,7 +137,9 @@ public void doTaxProcessing(String taxType, int reportYear, java.sql.Date startD
                     
                     // Do a smaller-batch retrieval last if necessary.
                     if (documentIds.size() % DOCID_MAX_BATCH_SIZE > 0) {
-                        selectStatement = con.prepareStatement(getForeignDraftAndWireTransferSelectSql(documentIds.size() % DOCID_MAX_BATCH_SIZE, summary));
+                    	String selectStatementSql = getForeignDraftAndWireTransferSelectSql(documentIds.size() % DOCID_MAX_BATCH_SIZE, summary, documentType);
+                    	LOG.info(selectStatementSql);

```

Comment:
This is logging at info level, even though the equivalent one above it is logging at debug level. It would be good to have the logging level consistent across both (probably debug).